cypress/integration/pages/onDemandRadio/testsForAMPOnly.js

```diff
@@ -12,7 +12,9 @@ export default ({ service, pageType }) => {
         cy.request(
           `${Cypress.env('currentPath')}.json${dataEndpointOverride()}`,
         ).then(({ body: jsonData }) => {
-          if (!isAvailable(jsonData)) {
+          if (!isAvailableOld(jsonData)) {
+            cy.log(`${Cypress.env('currentPath')}
+            `);
```

Comment:
We're already checking and logging the availablility within tests.js (which runs for AMP & Canonical), so this is not required here.